File: 19.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
rl.question("Enter type of fuel (1: Petrol, 2: Diesel, 3: Electric): ", function(input) {
    const x = parseInt(input);
    switch (x) {
        case 1:
            console.log("Petrol");
            break;
        case 2:
            console.log("Diesel");
            break;
        case 3:
            console.log("Electric");
            break;
    }
    rl.close();
});

File: 19.c
#include <stdio.h>

int main() 
{
    int x;
    printf("Enter fuel type of (1: Petrol, 2: Diesel, 3: Electric): ");
    scanf("%d", &x);
    switch (x) 
    {
        case 1:
            printf("Petrol\n");
            break;
        case 2:
            printf("Diesel\n");
            break;
        case 3:
            printf("Electric\n");
            break;
    }
    return 0;
}
